src/targets/gpu/fuse_ops.cpp

```diff
@@ -565,18 +565,21 @@ struct miopen_conv_bias
     template <class Self, class F>
     static auto reflect(Self& self, F f)
     {
+        assert(f.fp);
```

Comment:
The asserts should be in the methods to the `fusion` class not in the operators class.